README.md

# Wirdul latif Flutter Package

A Flutter package that provides access to the wirdul latif(morning and evening supplication by imam haddad) in  Arabic. This package allows you to retrieve the , Arabic wirds, and repetition count of each .


## Installation

To use this package, add it to your pubspec.yaml:

```bash
  dependencies:
  widul_latif: ^0.0.2

```

    
## Usage/Examples

Import the wirdul_latif library and use the provided functions to access the Names of Allah.

```javascript
import 'package:wirdul_latif/wirdul_latif.dart';

// Get the Morning Wird in Arabic (index between 1 - 44)
String morningWirdOne = getWirdArabic(index : 1);

// Get the Evening Wird in Arabic (index between 1 - 44)
String eveningWirdOne = getWirdArabic(index : 1,evening : true);

// Get the repitition count of every wird
String repCount = getWirdRepititionCount(index : 1);

// Get the length of wird list 
int wirdListLength = getWirdListLength();

```


## License

This package is released under the [MIT LICENSE](https://github.com/aslahmogral/wirdul_latif/blob/main/LICENSE). Feel free to use and modify it in your projects.

If you have any questions, issues, or suggestions, please open an issue on the GitHub repository.




## Author

This package is maintained by [@aslahmogral](https://github.com/aslahmogral).

Thank you for using the Wirdul Latif Flutter Package!





CHANGELOG.md
## 0.0.2

* formats fixed


## 0.0.1

* get morning and evening wird in arabic.
* get repition count of each wird.
* get length of wird list.


test/wirdul_latif_test.dart
import 'package:flutter_test/flutter_test.dart';
import 'package:wirdul_latif/wird_methods.dart';

void main() {
  test('get morning wird', () {
    expect(getWirdArabic(index: 13),
        "اللَّهُمَّ إِنِّي أَصْبَحْتُ مِنْكَ فِي نِعْمَةٍ وَعَافِيَةٍ وَسِتْرٍ، فَأَتِمَّ نِعْمَتَكَ عَلَيَّ وَعَافِيَتَكَ وَسِتْرَكَ فِي الدُّنْيَا وَالآخِرَةِ");
  });
  test('get evening wird', () {
    expect(getWirdArabic(index: 13, evening: true),
        "اللَّهُمَّ إِنِّي أَمْسَيْتُ مِنْكَ فِي نِعْمَةٍ وَعَافِيَةٍ وَسِتْرٍ، فَأَتِمَّ نِعْمَتَكَ عَلَيَّ وَعَافِيَتَكَ وَسِتْرَكَ فِي الدُّنْيَا وَالآخِرَةِ");
  });
  test('get wird count', () {
    expect(getWirdRepititionCount(index: 13),
        3);
  });

  test('get total  wird list lenght', () {
    expect(getWirdListLength(),
        44);
  });
}


example/lib/main.dart
import 'package:flutter/material.dart';
import 'package:wirdul_latif/wird_methods.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        // This is the theme of your application.
        //
        // Try running your application with "flutter run". You'll see the
        // application has a blue toolbar. Then, without quitting the app, try
        // changing the primarySwatch below to Colors.green and then invoke
        // "hot reload" (press "r" in the console where you ran "flutter run",
        // or simply save your changes to "hot reload" in a Flutter IDE).
        // Notice that the counter didn't reset back to zero; the application
        // is not restarted.
        primarySwatch: Colors.blue,
      ),
      home: const MyHomePage(title: 'Wirdul latif'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({super.key, required this.title});

  // This widget is the home page of your application. It is stateful, meaning
  // that it has a State object (defined below) that contains fields that affect
  // how it looks.

  // This class is the configuration for the state. It holds the values (in this
  // case the title) provided by the parent (in this case the App widget) and
  // used by the build method of the State. Fields in a Widget subclass are
  // always marked "final".

  final String title;

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  TextEditingController indexController = TextEditingController();
  String morningWird = '';
  String eveningWird = '';
  String WirdCount = '';

  @override
  Widget build(BuildContext context) {
    // This method is rerun every time setState is called, for instance as done
    // by the _incrementCounter method above.
    //
    // The Flutter framework has been optimized to make rerunning build methods
    // fast, so that you can just rebuild anything that needs updating rather
    // than having to individually change instances of widgets.
    return Scaffold(
      appBar: AppBar(
        // Here we take the value from the MyHomePage object that was created by
        // the App.build method, and use it to set our appbar title.
        title: const Text('Wirdul latif'),
      ),
      body: Center(
        // Center is a layout widget. It takes a single child and positions it
        // in the middle of the parent.
        child: Column(
          // Column is also a layout widget. It takes a list of children and
          // arranges them vertically. By default, it sizes itself to fit its
          // children horizontally, and tries to be as tall as its parent.
          //
          // Invoke "debug painting" (press "p" in the console, choose the
          // "Toggle Debug Paint" action from the Flutter Inspector in Android
          // Studio, or the "Toggle Debug Paint" command in Visual Studio Code)
          // to see the wireframe for each widget.
          //
          // Column has various properties to control how it sizes itself and
          // how it positions its children. Here we use mainAxisAlignment to
          // center the children vertically; the main axis here is the vertical
          // axis because Columns are vertical (the cross axis would be
          // horizontal).
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            Text('Enter the Index of Wird (between 1 to 44)'),
            Row(
              mainAxisSize: MainAxisSize.min,
              children: [
                Container(
                    width: MediaQuery.of(context).size.width / 2,
                    child: TextFormField(
                      controller: indexController,
                    )),
                ElevatedButton(
                    onPressed: () {
                      morningWird =
                          getWirdArabic(index: int.parse(indexController.text));
                      eveningWird = getWirdArabic(
                          index: int.parse(indexController.text),
                          evening: true);
                      WirdCount = getWirdRepititionCount(
                          index: int.parse(indexController.text));
                          setState(() {
                            
                          });
                    },
                    child: Text('Submit'))
              ],
            ),
            SizedBox(
              height: 28,
            ),
            Wrap(
              // mainAxisSize: MainAxisSize.min,
              children: [Text('Arabic Morning Wird : $morningWird')],
            ),
            Wrap(
              // mainAxisSize: MainAxisSize.min,
              children: [Text('Arabic Evening Wird : $eveningWird')],
            ),
            Wrap(
              // mainAxisSize: MainAxisSize.min,
              children: [
                Text('repeat Wird : $WirdCount')
              ],
            )
          ],
        ),
      ),
      // floatingActionButton: FloatingActionButton(
      //   onPressed: _incrementCounter,
      //   tooltip: 'Increment',
      //   child: const Icon(Icons.add),
      // ), // This trailing comma makes auto-formatting nicer for build methods.
    );
  }
}


lib/wird_methods.dart
library wirdul_latif;

import 'package:wirdul_latif/wird_text.dart';

List<int> eveningWirdIndex = [13, 14, 30, 31, 32, 33, 34, 35];

//get morning and evening wird in arabic
String getWirdArabic({required int index, bool evening = false}) {
  if (index <= 0 || index >= 44) {
    return 'please enter index between 1 and 44';
  }
  if (!evening) {
    return wirdText[index]["arabic"];
  } else {
    if (eveningWirdIndex.contains(index)) {
      return wirdText[index]["earabic"];
    } else {
      return wirdText[index]["arabic"];
    }
  }
}

//get wird count
String getWirdRepititionCount({required int index}) {
  if (index <= 0 || index >= 44) {
    return 'please enter index between 1 and 44';
  }
  return wirdText[index]['repetition'].toString();
}

//get the length of total list of wird
int getWirdListLength() {
  return 44;
}


lib/wird_text.dart
const Map wirdText = {
  1 :{
    "id": 1,
    "evening": "false",
    "repetition": 3,
    "arabic":
        "  قُلْ هُوَ اللهُ أَحَدٌ، اللّهُ الصَّمَدُ، لَمْ يَلِدْ وَلَمْ يُولَدْ،  وَلَمْ يَكُنْ لَهُ كُفُوًا أَحَدٌ "
  },
  2 :{
    "id": 2,
    "evening": "false",
    "repetition": 3,
    "arabic":
        "  قُلْ أَعُوذُ بِرَبِّ الْفَلَقِ، مِنْ شَرِّ مَا خَلَقَ، وَمِنْ شَرِّ غَاسِقٍ إِذَا وَقَبَ، وَمِنْ شَرِّ النَّفَّاثَاتِ فِي الْعُقَدِ، وَمِنْ شَرِّ حَاسِدٍ إِذَا حَسَدَ "
  },
 3 : {
    "id": 3,
    "evening": "false",
    "repetition": 3,
    "arabic":
        " قُلْ أَعُوذُ بِرَبِّ النَّاسِ، مَلِكِ النَّاسِ إِلَهِ النَّاسِ، مِنْ شَرِّ الْوَسْوَاسِ الْخَنَّاسِ الَّذِي يُوَسْوِسُ فِي صُدُورِ النَّاسِ، مِنَ الْجِنَّةِ وَالنَّاسِ "
  },
 4: {
    "id": 4,
    "evening": "false",
    "repetition": 3,
    "arabic":
        "رَبِّ أَعُوذُ بِكَ مِنْ هَمَزَاتِ الشَّيَاطِينِ، وَأَعُوذُ بِكَ رَبِّ أَنْ يَحْضُرُونِ "
  },
 5 : {
    "id": 5,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "أَفَحَسِبْتُم أَنَّمَا  خَلَقْنَاكُمْ عَبَثًا وَأَنَّكُمْ إِلَيْنَا لَا تُرْجَعُونَ، فَتَعَلَى اللَّهُ المَلِكُ الحَقُّ لَا إِلَهَ إِلَّا هُوَ رَبُّ العَرْشِ الكَرِيمِ، وَمَن يَدْعُ مَعَ اللَّهِ إِلَهَا آخَرَ لَا بُرْهَانَ لَهُ بِهِ فَإِنَّمَا حِسَابُهُ عِندَ رَبِّهِ إِنَّهُ لَا يُفْلِحُ الْكَافِرُونَ، وَقُلْ رَبِّ اغْفِرْ وَارْحَمْ وَأَنتَ خَيْرُ الرَّاحِمِينَ"
  },
  6 : {
    "id": 6,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "فسبحانَ اللهِ حِينَ تُمْسُونَ وَحِينَ تُصْبِحُونَ، وَلَهُ الْحَمْدُ فِي السَّمَواتِ وَالْأَرْضِ وَعَشِيَّا وَحِينَ تُظْهِرُونَ، يُخْرِجُ الْحَيَّ مِنَ الْمَيِّتِ وَيُخْرِجُ المَيِّتَ مِنَ الحَيِّ ويُحْيِي الأَرْضَ بَعْدَ مَوْتِهَا وَكَذَلِكَ تُخْرَجُونَ"
  },
 7:  {
    "id": 7,
    "evening": "false",
    "repetition": 3,
    "arabic":
        "أَعُوْذُ بِٱللّٰهِ ٱلسَّمِيعِ ٱلْعَلِيْمِ مِنَ ٱلشَّيْطَانِ ٱلرَّجِيمِ"
  },
 8 :  {
    "id": 8,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "لَوْ أَنْزَلْنَا هَٰذَا الْقُرْآنَ عَلَىٰ جَبَلٍ لَرَأَيْتَهُ خَاشِعًا مُتَصَدِّعًا مِنْ خَشْيَةِ اللَّـهِ ۚ وَتِلْكَ الْأَمْثَالُ نَضْرِبُهَا لِلنَّاسِ لَعَلَّهُمْ يَتَفَكَّرُونَ ﴿٢١﴾هُوَ اللَّـهُ الَّذِي لَا إِلَـٰهَ إِلَّا هُوَ ۖ عَالِمُ الْغَيْبِ وَالشَّهَادَةِ ۖ هُوَ الرَّحْمَـٰنُ الرَّحِيمُ ﴿٢٢﴾"
  },
  9: {
    "id": 9,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "هُوَ اللَّـهُ الَّذِي لَا إِلَـٰهَ إِلَّا هُوَ الْمَلِكُ الْقُدُّوسُ السَّلَامُ الْمُؤْمِنُ الْمُهَيْمِنُ الْعَزِيزُ الْجَبَّارُ الْمُتَكَبِّرُ ۚ سُبْحَانَ اللَّـهِ عَمَّا يُشْرِكُونَ ﴿٢٣﴾هُوَ اللَّـهُ الْخَالِقُ الْبَارِئُ الْمُصَوِّرُ ۖ لَهُ الْأَسْمَاءُ الْحُسْنَىٰ ۚ يُسَبِّحُ لَهُ مَا فِي السَّمَاوَاتِ وَالْأَرْضِ ۚ وَهُوَ الْعَزِيزُ الْحَكِيمُ."
  },
  10: {
    "id": 10,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "سلامٌ على نُوحٍ في العالَمين، إنَّا كذلكَ نَجْزِي المُحْسِنِينَ، إنَّهُ مِنْ عِبَادِنَا المُؤْمِنِينَ"
  },
  11: {
    "id": 11,
    "evening": "false",
    "repetition": 3,
    "arabic": "أَعُوذُ بِكَلِمَاتِ اللَّهِ التَّامَّاتِ مِنْ شَرِّ مَا خَلَقَ."
  },
  12: {
    "id": 12,
    "evening": "false",
    "repetition": 3,
    "arabic":
        "بِسْمِ اللهِ الذِي لَا يَضُرُّ مَعَ اسْمِهِ شَيْءٌ فِي الْأَرْضِ وَلَا فِي السَّمَاءِ، وَهُوَ السَّمِيعُ الْعَلِيمُ"
  },
  13: {
    "id": 13,
    "evening": "true",
    "repetition": 3,
    "arabic":
        "اللَّهُمَّ إِنِّي أَصْبَحْتُ مِنْكَ فِي نِعْمَةٍ وَعَافِيَةٍ وَسِتْرٍ، فَأَتِمَّ نِعْمَتَكَ عَلَيَّ وَعَافِيَتَكَ وَسِتْرَكَ فِي الدُّنْيَا وَالآخِرَةِ",
    "earabic":
        "اللَّهُمَّ إِنِّي أَمْسَيْتُ مِنْكَ فِي نِعْمَةٍ وَعَافِيَةٍ وَسِتْرٍ، فَأَتِمَّ نِعْمَتَكَ عَلَيَّ وَعَافِيَتَكَ وَسِتْرَكَ فِي الدُّنْيَا وَالآخِرَةِ"
  },
  14: {
    "id": 14,
    "evening": "true",
    "repetition": 4,
    "arabic":
        "اللَّهُمَّ، إِنِّي أَصْبَحْتُ أُشْهِدُكَ وَأُشْهِدُ حَمَلَةَ عَرْشِكَ وَمَلائِكَتَكَ وَجَمِيعَ خَلْقِكَ، أَنَّكَ أَنْتَ اللهُ، لا إِلهَ إِلَّا أَنْتَ، وَحْدَكَ لا شَرِيكَ لَكَ، وَأَنَّ مُحَمَّدًا عَبْدُكَ وَرَسُولُكَ.",
    "earabic":
        "اللَّهُمَّ، إِنِّي أَمْسَيْتُ أُشْهِدُكَ وَأُشْهِدُ حَمَلَةَ عَرْشِكَ وَمَلائِكَتَكَ وَجَمِيعَ خَلْقِكَ، أَنَّكَ أَنْتَ اللهُ، لا إِلَهَ إِلَّا أَنْتَ، وَحْدَكَ لا شَرِيكَ لَكَ، وَأَنَّ مُحَمَّدًا عَبْدُكَ وَرَسُولُكَ."
  },
  15: {
    "id": 15,
    "evening": "false",
    "repetition": 3,
    "arabic":
        "الْحَمْدُ لِلَّهِ رَبِّ الْعَالَمِينَ، حَمْدًا يُوَافِي نِعَمَهُ وَيُكَافِئُ مَزِيدَهُ."
  },
  16: {
    "id": 16,
    "evening": "false",
    "repetition": 3,
    "arabic":
        "آمَنْتُ بِاللهِ الْعَظِيمِ، وَكَفَرْتُ بِالْجِبْتِ وَالطَّاغُوتِ، وَاِسْتَمْسَكْتُ بِالْعُرْوَةِ الْوُثْقَى لاَ انفِصَامَ لَهَا، وَاللهُ سَمِيعٌ عَلِيمٌ."
  },
  17: {
    "id": 17,
    "evening": "false",
    "repetition": 3,
    "arabic":
        "رَضِيتُ بِاللَّهِ رَبًّا، وَبِالْإِسْلامِ دِينًا، وَبِسَيِّدِنَا مُحَمَّدٍ صَلَّى اللَّهُ عَلَيْهِ وَآلِهِ وَسَلَّمَ نَبِيًّا وَرَسُولًا."
  },
 18:  {
    "id": 18,
    "evening": "false",
    "repetition": 7,
    "arabic":
        "حَسْبِيَ اللَّهُ، لَا إِلَهَ إِلَّا هُوَ، عَلَيْهِ تَوَكَّلْتُ، وَهُوَ رَبُّ الْعَرْشِ الْعَظِيمِ."
  },
  19: {
    "id": 19,
    "evening": "false",
    "repetition": 10,
    "arabic":
        "اللَّهُمَّ صَلِّ وَسَلِّمْ عَلَى سَيِّدِنَا مُحَمَّدٍ، وَعَلَى آلِهِ وَصَحْبِهِ، وَسَلِّمْ."
  },
  20 :{
    "id": 20,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "اللَّهُمَّ إِنِّي أَسْأَلُكَ مِنْ فَجْأَةِ الْخَيْرِ، وَأَعُوذُ بِكَ مِنْ فَجْأَةِ الشَّرِّ."
  },
  21:{
    "id": 21,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "اللَّهُمَّ أَنْتَ رَبِّي لَا إِلَهَ إِلَّا أَنْتَ خَلَقْتَنِي، وَأَنَا عَبْدُكَ وَأَنَا عَلَى عَهْدِكَ وَوَعْدِكَ مَا اسْتَطَعْتُ، أَعُوذُ بِكَ مِنْ شَرِّ مَا صَنَعْتُ، أَبُوءُ لَكَ بِنِعْمَتِكَ عَلَيَّ وَأَبُوءُ بِذَنْبِي فَاغْفِرْ لِي فَإِنَّهُ لَا يَغْفِرُ الذُّنُوبَ إِلَّا أَنْتَ."
  },
  22:{
    "id": 22,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "اللَّهُمَّ أَنْتَ رَبِّي لا إِلَهَ إِلا أَنْتَ، عَلَيْكَ تَوَكَّلْتُ، وَأَنْتَ رَبُّ الْعَرْشِ الْعَظِيمِ، مَا شَاءَ اللَّهُ كَانَ، وَمَا لَمْ يَشَأْ لَمْ يَكُنْ، لا حَوْلَ وَلا قُوَّةَ إِلاَّ بِاللَّهِ الْعَلِيِّ الْعَظِيمِ، أَعْلَمُ أَنَّ اللَّهَ عَلَى كُلِّ شَيْءٍ قَدِيرٌ، وَأَنَّ اللَّهَ قَدْ أَحَاطَ بِكُلِّ شَيْءٍ عِلْمًا، اللَّهُمَّ إِنِّي أَعُوذُ بِكَ مِنْ شَرِّ نَفْسِي، وَمِنْ شَرِّ كُلِّ دَابَّةٍ أَنْتَ آخِذٌ بِنَاصِيَتِهَا، إِنَّ رَبِّي عَلَى صِرَاطٍ مُسْتَقِيمٍ."
  },
  23:{
    "id": 23,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "يَا حَيُّ يَا قَيُّومُ بِرَحْمَتِكَ أَسْتَغِيثُ، أَصْلِحْ لِي شَأْنِي كُلَّهُ، وَلَا تَكِلْنِي إِلَى نَفْسِي طَرْفَةَ عَيْنٍ."
  },
  24:{
    "id": 24,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "اللَّهُمَّ إِنِّي أَعُوذُ بِكَ مِنَ الْهَمِّ وَالْحَزَنِ، وَأَعُوذُ بِكَ مِنَ الْعَجْزِ وَالْكَسَلِ، وَأَعُوذُ بِكَ مِنَ الْجُبْنِ وَالْبُخْلِ، وَأَعُوذُ بِكَ مِنْ غَلَبَةِ الدَّيْنِ وَقَهْرِ الرِّجَالِ."
  },
  25:{
    "id": 25,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "اللَّهُمَّ إِنِّي أَسْأَلُكَ الْعَافِيَةَ فِي الدُّنْيَا وَالآخِرَةِ."
  },
  26:{
    "id": 26,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "اللَّهُمَّ إِنِّي أَسْأَلُكَ الْعَفْوَ وَالْعَافِيَةَ وَالْمَعَافَاةَ الدَّائِمَةَ فِي دِينِي وَدُنْيَايَ وَأَهْلِي وَمَالِي."
  },
  27:{
    "id": 27,
    "evening": "false",
    "repetition": 1,
    "arabic": "اللهمَّ اسْتُرْ عَوْرَاتِي، وَآمِنْ رَوْعَاتِي."
  },
  28:{
    "id": 28,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "اللَّهُمَّ، اِحْفَظْنِي مِنْ بَيْنِ يَدَيْ وَمِنْ خَلْفِي وَعَنْ يَمِينِي وَعَنْ شِمَالِي وَمِنْ فَوْقِي، وَأَعُوذُ بِعَظَمَتِكَ أَنْ أَغْتَالَ مِنْ تَحْتِي."
  },
  29:{
    "id": 29,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "اللَّهُمَّ، أَنْتَ خَلَقْتَنِي وَأَنْتَ تَهْدِينِي وَأَنْتَ تُطْعِمُنِي وَأَنْتَ تَسْقِينِي وَأَنْتَ تُمِيتُنِي وَأَنْتَ تُحْيِينِي وَأَنْتَ عَلَى كُلِّ شَيْءٍ قَدِيرٌ."
  },
  30:{
    "id": 30,
    "evening": "true",
    "repetition": 1,
    "arabic":
        "أَصْبَحْنَا عَلَى فِطْرَةِ الْإِسْلَامِ، وَعَلَى كَلِمَةِ الْإِخْلَاصِ، وَعَلَى دِينِ نَبِيِّنَا مُحَمَّدٍ صَلَّى اللَّهُ عَلَيْهِ وَآلِهِ وَسَلَّمَ، وَعَلَى مِلَّةِ أَبِينَا إِبْرَاهِيمَ حَنِيفًا مُسْلِمًا، وَمَا كَانَ مِنَ الْمُشْرِكِينَ.",
    "earabic":
        "أَمْسَيْنَا عَلَى فِطْرَةِ الْإِسْلَامِ، وَعَلَى كَلِمَةِ الْإِخْلَاصِ، وَعَلَى دِينِ نَبِيِّنَا مُحَمَّدٍ صَلَّى اللَّهُ عَلَيْهِ وَآلِهِ وَسَلَّمَ، وَعَلَى مِلَّةِ أَبِينَا إِبْرَاهِيمَ حَنِيفًا مُسْلِمًا، وَمَا كَانَ مِنَ الْمُشْرِكِينَ."
  },
  31:{
    "id": 31,
    "evening": "true",
    "repetition": 1,
    "arabic":
        "اللَّهُمَّ بِكَ أَصْبَحْنَا، وَبِكَ أَمْسَيْنَا، وَبِكَ نَحْيَا، وَبِكَ نَمُوتُ، وَعَلَيْكَ نَتَوَكَّلُ، وَإِلَيْكَ النُّشُورُ.",
    "earabic":
        "اللَّهُمَّ بِكَ أَمْسَيْنَا، وَبِكَ أَصْبَحْنَا، وَبِكَ نَحْيَا، وَبِكَ نَمُوتُ، وَعَلَيْكَ نَتَوَكَّلُ، وَإِلَيْكَ النُّشُورُ."
  },
  32:{
    "id": 32,
    "evening": "true",
    "repetition": 1,
    "arabic":
        "أَصْبَحْنَا وَأَصْبَحَ الْمُلْكُ لِلَّهِ، وَالْحَمْدُ لِلَّهِ رَبِّ الْعَالَمِينَ.",
    "earabic":
        "أَمْسَيْنَا وَأَمْسَى الْمُلْكُ لِلَّهِ، وَالْحَمْدُ لِلَّهِ رَبِّ الْعَالَمِينَ."
  },
  33:{
    "id": 33,
    "evening": "true",
    "repetition": 1,
    "earabic":
        "اللَّهُمَّ إِنِّي أَسْأَلُكَ خَيْرَ هَذَا اللَّيْلِ: فَتْحَهُ، وَنَصْرهُ، وَنُورَهُ، وَبَرَكَتَهُ، وَهُدَاهُ.",
    "arabic":
        "اللَّهُمَّ إِنِّي أَسْأَلُكَ خَيْرَ هَذَا الْيَوْمِ: فَتْحَهُ، وَنَصْرَهُ، وَنُورَهُ، وَبَرَكَتَهُ، وَهُدَاهُ."
  },
  34:{
    "id": 34,
    "evening": "true",
    "repetition": 1,
    "arabic":
        "اللَّهُمَّ إِنِّي أَسْأَلُكَ خَيْرَ هَذَا الْيَوْمِ وَخَيْرَ مَا قَبْلَهُ وَخَيْرَ مَا بَعْدَهُ، وَأَعُوذُ بِكَ مِنْ شَرِّ هَذَا الْيَوْمِ وَشَرِّ مَا بَعْدَهُ.",
    "earabic":
        "اللَّهُمَّ إِنِّي أَسْأَلُكَ خَيْرَ هَذَهِ اللَّيْلَةِ وَخَيْرَ مَا قَبْلَهَا وَخَيْرَ مَا بَعْدَهَا، وَأَعُوذُ بِكَ مِنْ شَرِّ هَذَهِ اللَّيْلَةِ وَشَرِّ مَا بَعْدَهَا."
  },
  35:{
    "id": 35,
    "evening": "true",
    "repetition": 1,
    "arabic":
        "اللَّهُمَّ مَا أَصْبَحَ بِي مِنْ نِعْمَةٍ أَوْ بِأَحَدٍ مِنْ خَلْقِكَ فَمِنْكَ وَحْدَكَ، لَا شَرِيكَ لَكَ، فَلَكَ الْحَمْدُ وَلَكَ الشُّكْرُ عَلَى ذَلِكَ.",
    "earabic":
        "اللَّهُمَّ مَا أَمْسَى بِي مِنْ نِعْمَةٍ أَوْ بِأَحَدٍ مِنْ خَلْقِكَ فَمِنْكَ وَحْدَكَ، لَا شَرِيكَ لَكَ، فَلَكَ الْحَمْدُ وَلَكَ الشُّكْرُ عَلَى ذَلِكَ."
  },
 36: {
    "id": 36,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "سُبْحَانَ اللَّهِ وَبِحَمْدِهِ، عَدَدَ خَلْقِهِ وَرِضَا نَفْسِهِ، وَزِنَةَ عَرْشِهِ، وَمِدَادَ كَلِمَاتِهِ."
  },
  37:{
    "id": 37,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "سُبْحَانَ اللَّهِ الْعَظِيمِ وَبِحَمْدِهِ، عَدَدَ خَلْقِهِ وَرِضَا نَفْسِهِ، وَزِنَةَ عَرْشِهِ، وَمِدَادَ كَلِمَاتِهِ."
  },
 38: {
    "id": 38,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "سُبْحَانَ اللَّهِ عَدَدَ مَا خَلَقَ فِي السَّمَاءِ، سُبْحَانَ اللَّهِ عَدَدَ مَا خَلَقَ فِي الْأَرْضِ، سُبْحَانَ اللَّهِ عَدَدَ مَا بَيْنَ ذَلِكَ، سُبْحَانَ اللَّهِ عَدَدَ مَا هُوَ خَالِقٌ."
  },
  39 :{
    "id": 39,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "الْحَمْدُ لِلَّهِ عَدَدَ مَا خَلَقَ فِي السَّمَاءِ، الْحَمْدُ لِلَّهِ عَدَدَ مَا خَلَقَ فِي الْأَرْضِ، الْحَمْدُ لِلَّهِ عَدَدَ مَا بَيْنَ ذَلِكَ، الْحَمْدُ لِلَّهِ عَدَدَ مَا هُوَ خَالِقٌ."
  },
  40:{
    "id": 40,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "لا إِلَٰهَ إِلاَّ اللَّٰهُ عَدَدَ مَا خَلَقَ فِي السَّمَاءِ، لا إِلَٰهَ إِلاَّ اللَّٰهُ عَدَدَ مَا خَلَقَ فِي الْأَرْضِ، لا إِلَٰهَ إِلاَّ اللَّٰهُ عَدَدَ مَا بَيْنَ ذَلِكَ، لا إِلَٰهَ إِلاَّ اللَّٰهُ عَدَدَ مَا هُوَ خَالِقٌ."
  },
  41:{
    "id": 41,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "اللهُ أَكْبَرُ عَدَدُ مَا خَلَقَ فِي السَّمَاءِ، اللهُ أَكْبَرُ عَدَدُ مَا خَلَقَ فِي الْأَرْضِ، اللهُ أَكْبَرُ عَدَدُ مَا بَيْنَ ذَلِكَ، اللهُ أَكْبَرُ عَدَدُ مَا هُوَ خَالِقٌ."
  },
  42:{
    "id": 42,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "لاَ حَوْلَ وَلاَ قُوَّةَ إِلاَّ بِاللَّهِ الْعَلِيِّ الْعَظِيْمِ، عَدَدَ مَا خَلَقَ فِي السَّمَاءِ، وَلاَ حَوْلَ وَلاَ قُوَّةَ إِلاَّ بِاللَّهِ الْعَلِيِّ الْعَظِيْمِ، عَدَدَ مَا خَلَقَ فِي الأَرْضِ، وَلاَ حَوْلَ وَلاَ قُوَّةَ إِلاَّ بِاللَّهِ الْعَلِيِّ الْعَظِيْمِ، عَدَدَ مَا بَيْنَ ذَلِكَ، وَلاَ حَوْلَ وَلاَ قُوَّةَ إِلاَّ بِاللَّهِ الْعَلِيِّ الْعَظِيْمِ، عَدَدَ مَا هُوَ خَالِقٌ"
  },
  43:{
    "id": 43,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "لا إلَهَ إِلَّا اللَّهُ وَحْدَهُ لَا شَرِيكَ لَهُ، لَهُ الْمُلْكُ وَلَهُ الْحَمْدُ يُحْيِي وَيُمِيتُ وَهُوَ عَلَى كُلِّ شَيْءٍ قَدِيرٌ، عَدَدَ دُ كُلِّ ذَرَّةٍ أَلْفَ مَرَّةٍ."
  },
  44:{
    "id": 44,
    "evening": "false",
    "repetition": 1,
    "arabic":
        "اللَّهُمَّ صَلِّ وَسَلِّمْ عَلَى سَيِّدِنَا مُحَمَّدٍ مَفْتَاحِ بَابِ رَحْمَةِ اللَّهِ عَدَدَ مَا فِي عِلْمِ اللَّهِ، صَلَاةً وَسَلَامًا دَائِمِيْنَ بِدَوَامِ مُلْكِ اللَّهِ، وَعَلَى آلِهِ وَصَحْبِهِ، عَدَدَ كُلِّ ذَرَّةٍ أَلْفَ مَرَّةٍ."
  }
};
